src/frontend/web_application/src/components/OpenPGPKey/index.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { Component, PropTypes } from 'react';
+import { enums } from 'openpgp';
+import classnames from 'classnames';
+import { DateTime, withTranslator } from '@gandi/react-translate';
+import Button from '../Button';
+import Icon from '../Icon';
+import DefList from '../DefList';
+import { getKeyFromASCII } from '../../services/openpgp-manager';
+import './style.scss';
+
+function getViewValues(publicKey, keyStatuses) {
+  return {
+    fingerprint: publicKey.primaryKey.fingerprint,
+    userId: publicKey.getPrimaryUser().user.userId.userid,
+    createdAt: publicKey.primaryKey.created,
+    expirationTime: publicKey.getExpirationTime(),
+    algorithm: publicKey.primaryKey.algorithm,
+    bitSize: publicKey.primaryKey.getBitSize(),
+    userIds: publicKey.users.map(user => user.userId.userid),
+    keyStatus: Object.keys(keyStatuses)
+      .find(statusLiteral => keyStatuses[statusLiteral] === publicKey.verifyPrimaryKey()),
+  };
+}
+
+@withTranslator()
```

Comment:
If I'm sure this kind of component is not shared, it will be easily moved to "scenes" (aka routed components) But it is not always the case. 
So I don't know how to to this.